examples/siro_sandbox/server/server.py

```diff
@@ -0,0 +1,188 @@
+import asyncio
+import json
+import os
+import ssl
+import threading
+
+import websockets
+
+from . import multiprocessing_config
+from .frequency_limiter import FrequencyLimiter
+
+# Boolean variable to indicate whether to use SSL
+use_ssl = False
+
+server_process = None
+exit_event = None
+
+
+def launch_server_process(interprocess_record):
+    # see multiprocessing_config to switch between real and dummy multiprocessing
+
+    global server_process
+    global exit_event
+
+    # multiprocessing.dummy.Process is actually a thread and requires special logic
+    # to terminate it.
+    exit_event = (
+        threading.Event() if multiprocessing_config.use_dummy else None
+    )
+
+    server_process = multiprocessing_config.Process(
+        target=server_main, args=(interprocess_record, exit_event)
+    )
+    server_process.start()
+
+
+def terminate_server_process():
+    global server_process
+    global exit_event
+    if multiprocessing_config.use_dummy:
+        if exit_event:
+            exit_event.set()
+            exit_event = None
+    else:
+        if server_process:
+            server_process.terminate()
+            server_process = None
+
+
+def create_ssl_context():
+    ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
+    certfile = "./examples/siro_sandbox/self_signed.pem"
```

Comment:
(No need to generate these files for the Unity VR client/server. It doesn't use SSL.)